modules/layers/src/column-layer/column-layer.js

```diff
@@ -148,24 +180,47 @@ export default class ColumnLayer extends Layer {
         i++;
       }
     }
-    model.setProps({geometry});
+
+    this.state.models.forEach(model => {
+      if (model) {
+        model.setProps({geometry});
+      }
+    });
   }
 
   draw({uniforms}) {
-    const {elevationScale, extruded, offset, coverage, radius, angle} = this.props;
-
-    this.state.model
-      .setUniforms(
-        Object.assign({}, uniforms, {
-          radius,
-          angle: (angle / 180) * Math.PI,
-          offset,
-          extruded,
-          coverage,
-          elevationScale
-        })
-      )
-      .draw();
+    const {
+      elevationScale,
+      wireframe,
+      filled,
+      extruded,
+      offset,
+      coverage,
+      radius,
+      angle
+    } = this.props;
+    const {filledModel, strokeModel} = this.state;
+    const renderUniforms = Object.assign({}, uniforms, {
+      radius,
+      angle: (angle / 180) * Math.PI,
+      offset,
+      extruded,
+      coverage,
+      elevationScale
+    });
+    const numInstances = this.getNumInstances();
+    if (strokeModel && wireframe) {
+      strokeModel.setInstanceCount(numInstances);
+      strokeModel.setUniforms(renderUniforms);
+      strokeModel.setDrawMode(GL.LINES);
+      strokeModel.setUniforms({isWireframe: true}).draw();
```

Comment:
`drawMode` and this uniform can be set once in `getModels`

Reply:
Planning to reuse the same one model, the way you have advised. So, `drawMode` could be set in `draw` itself for various conditions.

Reply:
I had to have separate models as they had different `drawModes` and the `indices` generated for each model was a bit different.